src/reportportal/config.py

```
@@ -43,20 +43,21 @@ def load_config_file() -> Dict[str, Any]:
 
     try:
         with open(config_file, 'r') as f:
             config = yaml.safe_load(f)
             if config is None:
-                logger.debug(f"Config file is empty: {config_file}")
+                logger.debug("Config file empty")
                 return {}
-            logger.debug(f"Loaded config from: {config_file}")
+            logger.debug("Config file loaded successfully")
             return config
     except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)